components/identity-core/org.wso2.carbon.identity.core/src/main/java/org/wso2/carbon/identity/core/internal/DefaultServiceURLBuilder.java

```diff
@@ -418,7 +442,7 @@ public String getRelativeInternalURL() {
             return relativeInternalUrl;
         }
 
-        private String fetchAbsoluteUrl() throws URLBuilderException {
+        private String fetchAbsolutePublicUrl() throws URLBuilderException {
```

Comment:
Shall we change the below variable name also to `absolutePublicUrl`?

Reply:
Fixed https://github.com/wso2/carbon-identity-framework/pull/2949/commits/718c68995f4548bc59738aff6b21337e7d8164ea